packages/react-three-rapier/src/Attractor.tsx

```diff
@@ -0,0 +1,62 @@
+import * as THREE from 'three'
+import { useFrame } from "@react-three/fiber";
+import { RigidBody } from '@dimforge/rapier3d-compat';
+import { Vector3Array } from './types';
+import { useRapier } from './hooks';
+type GravityType = "static" | "linear" | "newtonian";
+interface AttractorProps {
+  position?: Vector3Array;
+  strength?: number;
+  range?: number;
+  showHelper?: boolean;
+  gravityType?: GravityType;
+  gravitationalConstant?: number;
+}
+export const Attractor = ({
+  position = [0, 0, 0],
+  strength = 0.5,
+  range = 15,
+  showHelper = false,
+  gravityType = 'static',
+  gravitationalConstant = 6.673e-11,
+}: AttractorProps) => {
+  const { world } = useRapier()
+  const gravitySource = new THREE.Vector3(position[0], position[1], position[2])
```

Comment:
It's just a smaller point, but all these constants including the function below will be recreated every time this component re-renders. This should ideally not happen a lot _anyway_, but no reason to be wasteful.